app/models/location.rb

```diff
@@ -108,6 +108,10 @@ def reactivating_soon?
     status == :reactivating_soon
   end
 
+  def deactivates_in_more_than_6_months?
+    status == :deactivating_soon && open_deactivation.present? && open_deactivation.deactivation_date > 6.months.from_now
+  end
```

Comment:
I'd make this more generic:

```suggestion
  DEACTIVATING_SOON_CUTOFF = 6.months
  
  def deactivates_in_a_long_time?
    status == :deactivating_soon && open_deactivation.present? && open_deactivation.deactivation_date > DEACTIVATING_SOON_CUTOFF.from_now
  end
```

But, is this the same as?

```suggestion  
  def deactivates_in_a_long_time?
    status_at(6.months.from_now) == :deactivating_soon
  end
```

Reply:
Maybe, although I can imagine there being some weird situations going on when there are multiple deactivation periods in the future. I'll see if it works tho, thanks!